apps/accounts/risk_handlers.py

```diff
@@ -174,10 +188,10 @@ def handle_change_password_add(self):
             'check_conn_after_change': True,
             "name": "Push account password: {}@{}".format(self.username, asset.name),
         }
-        execution.save()
-        execution.start()
+        self.start_execution(execution)
 
-        GatheredAccount.objects.filter(asset=self.asset, username=self.username).update(
-            present=True
+        (
+            GatheredAccount.objects
+            .filter(asset=self.asset, username=self.username)
+            .update(present=True)
         )
-        return execution.summary
```

Comment:
I am sorry but I can't analyze or provide feedback on your code at this moment since you haven't provided it. However, please don't hesitate to share your code; all information such as file paths and line numbers would be helpful in reviewing.

If you need any assistance with something specific like coding best practices, how certain features of Django work, etc., I'm here to help! Just mention what aspects of the current content interests you most so that I can tailor my response accordingly.